waffle-compiler/test/projects/defaultConfigFile/contracts/Constantinople.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.1;
+
+
+contract Constantinople {
```

Comment:
Let's use some other example with solidity 0.6.* and nothing to do with constantinopole